src/WebJobs.Script/Description/Binding/ApiHubBindingMetadata.cs

```diff
@@ -14,6 +14,8 @@ public class ApiHubBindingMetadata : BindingMetadata
         [JsonIgnore]
         public string Key { get; set; }
 
+        public int PollIntervalInSeconds { get; set; }
```

Comment:
You don't apply [JsonIgnore] on this one, but you do on the one above. I'm wondering why the above ignore attribute even exists - I don't see any of the other metadata classes using this.